cmds/ls/fileinfo_linux.go

```diff
@@ -106,14 +107,33 @@ type quotedStringer struct {
 }
 type longStringer struct {
 	fileInfo
-	comp fmt.Stringer // decorator pattern
+	comp  fmt.Stringer // decorator pattern
+	human bool
 }
 
 // Return the name surrounded by quotes with escaped control characters.
 func (fi quotedStringer) String() string {
 	return fmt.Sprintf("%#v", fi.name)
 }
 
+func sizeToString(size int64) string {
```

Comment:
This looks good to me, but what about something like https://github.com/dustin/go-humanize
?